main.cpp

```diff
@@ -630,6 +630,8 @@ HashInfo g_hashes[] =
   {0x0} /* !! and many more. too many */ },
 { mum_low_test,         32, MUMLOW_VERIF,"MUMlow",     "github.com/vnmakarov/mum-hash", GOOD,
   {0x11fb062a, 0x3ca9411b, 0x3edd9a7d, 0x41f18860, 0x691457ba} /* !! */ },
+{ mum_v3_hash_test,     64, MUM_VERIF,  "MUMv3",       "github.com/vnmakarov/mum-hash", POOR,
```

Comment:
Should be 0x8BD72B8C

Reply:
I am not familiar with the secrets. 
Do you mean `{0x0, 0x8bd72b8c}`? Changed.

Reply:
You need to find out why the hash different on different platforms, and how. And add this value then, so that the verification test passes